xmake/modules/package/manager/pacman/find_package.lua

```diff
@@ -23,6 +23,76 @@ import("core.base.option")
 import("lib.detect.find_tool")
 import("package.manager.pkgconfig.find_package", {alias = "find_package_from_pkgconfig"})
 
+-- get result from list of file inside pacman package
+function _find_package_from_list(list, opt, cygpath)
+    local result = {
+        includedirs = {},
+        linkdirs = {},
+        links = {},
+        version = nil
+    }
```

Comment:
we need return nil if not found, so we need define `local result = nil` first

Reply:
I added `return nil` when the list, cygpath or the version are not found.